.goreleaser.yaml

```diff
@@ -0,0 +1,287 @@
+# Documentation at https://goreleaser.com
+version: 2
+project_name: editorconfig-checker
+env:
+  - REPO_LICENSE=MIT
+  - DOCKERHUB_USERNAME={{ if index .Env "DOCKERHUB_USERNAME"  }}{{ .Env.DOCKERHUB_USERNAME }}{{ else }}echoix{{ end }}
+  - GHCR_USERNAME={{ if index .Env "GHCR_USERNAME"  }}{{ .Env.GHCR_USERNAME }}{{ else }}echoix{{ end }}
+before:
+  hooks:
+    - echo {{.Version}} > VERSION
+    - go mod tidy
+    - go generate ./...
+builds:
+  - id: "editorconfig-checker"
+    main: ./cmd/editorconfig-checker
```

Comment:
While it would be best, I have not found a way to rename or copy a binary in Goreleaser's DSL.
I kept both builds, so that we can publish archives and images in the new style parallel to the ones in the old style - so we can even during the lifetime of v3.1 go around and adjust consumers of our images and archives, instead of having to do it right when v4.0 comes out.